### ISSUE TITLE
deprecation

### ISSUE THREAD

**Issue by dltacube** [BurntSushi/goim#10]
Hello,

I'm wondering if you had any plans on updating this project when it takes its new form or if you were ceasing development completely.

**Comment by BurntSushi** [comment]
No. This specific project will die. Most of the complexity in this code is in the parsing work. Moreover, the "new" data set won't contain nearly as much data as the old data set, so a lot of the work around organizing the various data types will be obsolete as well. Moreover, the new data is in CSV format, which makes it a lot more accessible. The old format was arcane and ridiculously hard to parse in a uniform way, but the CSV data is fairly well organized such that anyone with basic Python knowledge can use it effectively.

Note that goim should continue to work so long as the old data is up. As of now, they've extended the date to December 28, 2017.

@heyoni Out of curiosity, what are you using goim for?

**Comment by dltacube** [comment]
That makes sense. Hopefully they retain the new format without shrinking the available data so badly.

I've been using goim to create a web app that would notify me of any new releases I might be interested based on individuals or franchises I follow. I'm doing this for music, movies and TV so far using musicbrainz as my music repository but it looks like I may have to switch to something else since imdb isn't including release dates with their new format.
